### PR TITLE
[samples] Add support for 12bit color formats

### PR DESCRIPTION
**Pull request by egorovdanil**

### CODE REVIEW

**Comment by egorovdanil** [comment]
Test:
```
sample_vpp -scc p016 -dcc y216 -sw 1920 -sh 1080 -i p016.yuv -o out.yuv
sample_vpp -scc y216 -dcc y416 -sw 1920 -sh 1080 -i y216.yuv -o out.yuv
sample_vpp -scc y416 -dcc p016 -sw 1920 -sh 1080 -i y416.yuv -o out.yuv
sample_decode h265 -p016 -i 12b_420.bit -o out.yuv
sample_decode h265 -y216 -i 12b_422.bit -o out.yuv
sample_decode h265 -y416 -i 12b_444.bit -o out.yuv
sample_multi_transcode -i::h265 12b_420.bit -dc::p016 -ec::p016 -o::h265 out.h265
sample_multi_transcode -i::h265 12b_422.bit -dc::y216 -ec::y216 -o::h265 out.h265
sample_multi_transcode -i::h265 12b_444.bit -dc::y416 -ec::y216 -o::h265 out.h265
```

File: samples/sample_decode/src/pipeline_decode.cpp [hunk 1]
```diff
@@ -1034,6 +1034,11 @@ mfxStatus CDecodingPipeline::InitVppParams()
         (  m_mfxVppVideoParams.vpp.Out.FourCC == MFX_FOURCC_P010
 #if (MFX_VERSION >= 1027)
         || m_mfxVppVideoParams.vpp.Out.FourCC == MFX_FOURCC_Y210
+#endif
+#if (MFX_VERSION >= 1330)
```
Comment (dmitryermilov) [inline samples/sample_decode/src/pipeline_decode.cpp:1038]: 1330?

**Review by dmitryermilov — CHANGES_REQUESTED** [review]

File: samples/sample_decode/src/pipeline_decode.cpp (hunk 1 above)
Comment (egorovdanil) [reply samples/sample_decode/src/pipeline_decode.cpp]: corrected to 1331

Comment (dmitryermilov) [reply samples/sample_decode/src/pipeline_decode.cpp]: No. https://github.com/Intel-Media-SDK/MediaSDK/blob/c8ab21caa250934b456a744d8dd90ae75285f9c9/api/include/mfxstructures.h#L107-L109

Comment (egorovdanil) [reply samples/sample_decode/src/pipeline_decode.cpp]: corrected to 1031

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**